lawnchair/src/app/lawnchair/search/LawnchairAppSearchAlgorithm.kt

```diff
@@ -107,13 +120,23 @@ class LawnchairAppSearchAlgorithm(context: Context) : LawnchairSearchAlgorithm(c
 
         return apps.asSequence()
             .filter { StringMatcherUtility.matches(queryTextLower, it.title.toString(), matcher) }
+            .filter {
+                showHiddenAppsInSearchBar || !hiddenApps.stream()
+                    .anyMatch() { s -> s.contains(it.componentName.flattenToString()) }
```

Comment:
Use [`Set.contains`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.collections/-set/contains.html) instead.

Reply:
Yes, that's right, it would be `it.componentName.flattenToString() !in hiddenApps`.